common.go

```diff
@@ -515,68 +515,68 @@ func getDecodedResponseMultiGet(response objectsMap) map[string]bool {
 }
 
 // GetElasticTemplatesAndPolicies will return elastic templates and policies
-func GetElasticTemplatesAndPolicies(path string, useKibana bool) (map[string]*bytes.Buffer, map[string]*bytes.Buffer, error) {
+func GetElasticTemplatesAndPolicies(useKibana bool) (map[string]*bytes.Buffer, map[string]*bytes.Buffer, error) {
```

Comment:
Please remove this from common. It's place should be in a factory-named file as it is a factory method.

Reply:
This refactor was done in the next PR.

Reply:
ok